ci/scripts/pr_list_database.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python3
 
 import sys
+import os
 from pathlib import Path
-from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, REMAINDER
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, REMAINDER, ZERO_OR_MORE
 import sqlite3
 
 
-def sql_connection(filename: Path) -> sqlite3.Connection:
+def dir_path(string):
```

Comment:
As dumb as it seems for so simple a function, there should have a doc-block for this.

Reply:
Since this will return the absolute path for a file as well, should it be named `abs_path` of `full_path` instead?

Reply:
This was to make sure it was one or the other and nothing else, I guess you are asking me to over document and split this into two types.

Reply:
Oh wait, I see now, you are asking to simply change the name of the function, yes that makes more sense, will do.

Reply:
Yeah, those were two different things about the same function:
- It needs a doc-block
- If it handles more than just directory paths, the name shouldn't imply it is only for directories

Reply:
- Why does this need a doc block, doesn't that defeat the purpose of useful documentation?
- I did rename it.

Reply:
Because every function/method should have a doc-block. When we finally get around to parsing these to create API documentation, it will be needed. It should be very straightforward and short.